include/openssl/core_names.h

```diff
@@ -49,6 +49,7 @@ extern "C" {
 #define OSSL_CIPHER_PARAM_AEAD_TLS1_AAD_PAD  "tlsaadpad"  /* size_t */
 #define OSSL_CIPHER_PARAM_AEAD_TLS1_IV_FIXED "tlsivfixed" /* octet_string */
 #define OSSL_CIPHER_PARAM_AEAD_IVLEN         "aeadivlen"  /* size_t */
+#define OSSL_CIPHER_PARAM_RANDOM_KEY         "randkey"    /* octet_string */
```

Comment:
Please document this in doc/man7/provider-cipher.pod

Reply:
You only just added that file :)

Reply:
doc updated.

Reply:
> You only just added that file :)

Just trying to keep you on your toes :-)